staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc.go

```diff
@@ -125,6 +126,7 @@ func newOIDCAuthProvider(clusterAddress string, cfg map[string]string, persister
 	}
 
 	if len(cfg[cfgExtraScopes]) > 0 {
+		//nolint:logcheck // Should not happen.
 		klog.V(2).Infof("%s auth provider field depricated, refresh request don't send scopes",
```

Comment:
This can totally happen so we should update it.

Reply:
But only if used incorrectly, right?

Do you prefer structured logging here?